src/main/kotlin/org/rust/ide/inspections/lints/RsUnknownCrateTypesInspection.kt

```diff
@@ -41,6 +41,6 @@ class RsUnknownCrateTypesInspection : RsLintInspection() {
         }
 
     companion object {
-        private val KNOWN_CRATE_TYPES = listOf("bin", "lib", "dylib", "staticlib", "cdylib", "rlib", "proc-macro")
+        val KNOWN_CRATE_TYPES = listOf("bin", "lib", "dylib", "staticlib", "cdylib", "rlib", "proc-macro")
```

Comment:
Maybe there can be a better place for this constant, as it now used in both inspection and completion